app/views/smart_answers/result.html.erb

```diff
@@ -28,20 +28,6 @@
         } unless outcome.title_as_heading? %>
       <% end %>
 
-      <% if @presenter.hide_previous_answers_on_results_page? %>
-        <p class="govuk-body govuk-!-margin-bottom-6">
-          <%= link_to "Start again",
-            restart_flow_path(@presenter),
-            :class => "govuk-link",
-            :data => {
-              module: "track-click",
-              "track-action": "Start again",
-              "track-category": "StartAgain",
-              "track-label": @presenter.current_node.title
-            } %>
-        </p>
-      <% end %>
-
```

Comment:
Do we need to remove the changing merging on line 20 too?

Reply:
Yes, that's superfluous now.  Done